docs/dml-operation-on-carbondata.md

```diff
@@ -149,6 +149,50 @@ You can use the following options to load data:
    
    * If this option is set to TRUE, then high.cardinality.identify.enable property will be disabled during data load.
    
+- **SORT_SCOPE:** This property can have four possible values :
+
+    * BATCH_SORT : The sorting scope is smaller and more index tree will be created,thus loading is faster but query maybe slower.
```

Comment:
mention this is sort based on memory size, and carbon will create one index for each batch